src/plugins/management/public/plugin.tsx

```diff
@@ -167,6 +172,49 @@ export class ManagementPlugin
       });
     }
 
+    // Register the Settings app only if in serverless, until we integrate the SettingsApplication into the Advanced settings plugin
+    // Otherwise, it will be double registered from the Advanced settings plugin
+    if (_plugins.serverless) {
+      const title = kbnI18n.translate('management.settings.settingsLabel', {
+        defaultMessage: 'Advanced Settings',
+      });
+
+      this.managementSections.definedSections.kibana.registerApp({
+        id: 'settings',
+        title,
+        order: 3,
+        async mount({ element, setBreadcrumbs }) {
+          setBreadcrumbs([{ text: title }]);
+          const {
+            i18n,
+            docLinks,
+            notifications: { toasts },
+            settings,
+            theme,
+          } = core;
+
+          ReactDOM.render(
+            <KibanaRenderContextProvider {...core}>
+              <SettingsApplicationKibanaProvider
+                settings={{
+                  client: settings.client,
+                }}
+                theme={theme}
+                i18nStart={i18n}
```

Comment:
this doesn't look right... there are too many props/repeated props.  I think this should be:

```
<SettingsApplicationKibanaProvider {...{ settings, toasts, docLinks, theme, i18nStart }} />
```

nit: I think `i18nStart` should be renamed `i18n`, as well, in the `form` package.